app/controllers/stripe_active_cards_controller.rb

```diff
@@ -11,16 +11,14 @@ def create
       Rails.logger.info("Stripe Add New Card Success - #{current_user.username}")
       flash[:settings_notice] = "Your billing information has been updated"
     else
-      DatadogStatsClient.increment("stripe.errors")
+      DatadogStatsClient.increment("stripe.errors.new_subscription")
```

Comment:
Let's add these descriptors as tags that way its easier to group the errors in Datadog. Also we pay by the metric name so thats another reason to try and group things. Something like this. You could also add user_id to the tags if you wanted.
```ruby
DatadogStatsClient.increment("stripe.errors", tags: ["event:new_subscription"])
```